frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -111,14 +129,22 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
               aria-label="Clone Pvc"
             />
           </FormGroup>
-          <FormGroup label="Size" isRequired fieldId="clone-pvc-modal__size">
+          <FormGroup
+            label="Size"
+            isRequired
+            fieldId="clone-pvc-modal__size"
+            className="co-clone-pvc-modal__ocs-size"
+            helperTextInvalid="Size should be equal or higher than default size"
+            validated={validSize ? 'default' : 'error'}
+          >
             <RequestSizeInput
               name="requestSize"
               testID="input-request-size"
               onChange={requestedSizeInputChange}
               defaultRequestSizeUnit={requestedUnit}
               defaultRequestSizeValue={requestedSize}
               dropdownUnits={dropdownUnits}
+              isInputDisabled={isCephProvisioner(scResource?.provisioner)}
```

Comment:
You should wait for scResource to load, otherwise on slower connections you may end up with Size input enabled for some time.